app/controllers/DepartureRejectionController.scala

```diff
@@ -23,20 +23,22 @@ import utils.JsonUtils
 
 class DepartureRejectionController @Inject()(cc: ControllerComponents, jsonUtils: JsonUtils) extends BackendController(cc) {
 
-  private val GuaranteeNotValidDepartureId: Int        = 27
-  private val GuaranteeNotValidMessageId: Int          = 2
-  private val DeclarationRejectionDepartureId: Int     = 33
-  private val DeclarationRejectionMessageId: Int       = 2
-  private val CancellationDecisionUpdateId: Int        = 39
-  private val CancellationDecisionUpdateMessageId: Int = 2
-  private val DeclarationCancellationId: Int           = 45
-  private val DeclarationCancellationMessageId: Int    = 2
-  private val NoReleaseForTransitId: Int               = 31
-  private val NoReleaseForTransitMessageId: Int        = 2
-  private val ControlDecisionId: Int                   = 32
-  private val ControlDecisionMessageId: Int            = 2
-  private val CancellationRequestDepartureId: Int      = 23
-  private val MRNAllocatedMessageId: Int               = 2
+  private val GuaranteeNotValidDepartureId: Int                 = 27
+  private val GuaranteeNotValidMessageId: Int                   = 2
+  private val DeclarationRejectionDepartureId: Int              = 33
+  private val DeclarationRejectionMessageId: Int                = 2
+  private val CancellationDecisionUpdateId: Int                 = 39
+  private val CancellationDecisionUpdateMessageId: Int          = 2
+  private val CancellationDecisionUpdateRejectionId: Int        = 88
+  private val CancellationDecisionUpdateMessageRejectionId: Int = 2
+  private val DeclarationCancellationId: Int                    = 45
+  private val DeclarationCancellationMessageId: Int             = 2
+  private val NoReleaseForTransitId: Int                        = 31
+  private val NoReleaseForTransitMessageId: Int                 = 2
+  private val ControlDecisionId: Int                            = 32
+  private val ControlDecisionMessageId: Int                     = 2
+  private val CancellationRequestDepartureId: Int               = 23
+  private val MRNAllocatedMessageId: Int                        = 2
```

Comment:
Minor suggestion but could these be popped in a companion object to avoid the need to duplicate them in the spec?